packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5443,26 +5423,34 @@ export default class Meeting extends StatelessWebexPlugin {
 
       switch (event.state) {
         case ConnectionState.Connecting:
-          // @ts-ignore
-          this.webex.internal.newMetrics.submitClientEvent({
-            name: 'client.ice.start',
-            options: {
-              meetingId: this.id,
-            },
-          });
+          if (this.networkStatus !== NETWORK_STATUS.DISCONNECTED) {
```

Comment:
Might use ```hasMediaConnectionConnectedAtLeastOnce``` here instead once https://github.com/webex/webex-js-sdk/pull/3292 goes in

Reply:
Done